Started UtBot Contest, classfileDir = /var/benchmarks/projects/guava/guava/target/classes, classpathString=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, outputDir=/home/utbot_mocks/temp/testcases, mocks=Other classes: Mockito
18:31:22.200 | INFO  | Started: warmup: kotlin reflection :: init
18:31:22.273 | INFO  | Finished (in 0.073 sec): warmup: kotlin reflection :: init 
[INIT]
>> [RUN] com.google.common.base.Throwables 30
18:31:22.281 | DEBUG | -----------------------------------------------------------------------------
18:31:22.282 | INFO  | Contest.runGeneration: Time budget: 30000 ms, Generation timeout=25500 ms, classpath=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, methodNameFilter=null
18:31:22.283 | INFO  | ClassUnderTest[ FQN: com.google.common.base.Throwables
    classfileDir: /var/benchmarks/projects/guava/guava/target/classes
    testClassSimpleName: ThrowablesTest
    generatedTestFile: /home/utbot_mocks/temp/testcases/com/google/common/base/ThrowablesTest.java
    generatedTestsSourcesDir: /home/utbot_mocks/temp/testcases
]
18:31:22.304 | INFO  | Started: class com.google.common.base.Throwables
18:31:22.305 | INFO  | Started: preparation class class com.google.common.base.Throwables: kotlin reflection :: run
18:31:22.338 | INFO  | Finished (in 0.033 sec): preparation class class com.google.common.base.Throwables: kotlin reflection :: run 
18:31:22.339 | INFO  | Started: 2nd optional soot initialization
18:31:22.340 | INFO  | Finished (in 0.001 sec): 2nd optional soot initialization 
18:31:22.349 | DEBUG |  ... 
18:31:22.365 | INFO  | Started: method com.google.common.base.Throwables.getRootCause(Throwable)
18:31:22.366 | INFO  |  -- Remaining time budget: 25432 ms, #remaining_methods: 20, budget for method: 1271 ms, solver timeout: 635 ms, budget for symbolic execution: 636 ms, budget for concrete execution: 635 ms,  -- 
18:31:22.710 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)1
18:31:22.992 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getRootCause(Throwable)
18:31:23.059 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)2
18:31:23.358 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)3
18:31:23.629 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getRootCause(Throwable)
18:31:23.809 | INFO  | Finished (in 1.443 sec): method com.google.common.base.Throwables.getRootCause(Throwable) 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
18:31:23.822 | DEBUG |  ... 
18:31:23.822 | INFO  | Started: method com.google.common.base.Throwables.propagate(Throwable)
18:31:23.823 | INFO  |  -- Remaining time budget: 23961 ms, #remaining_methods: 19, budget for method: 1261 ms, solver timeout: 630 ms, budget for symbolic execution: 631 ms, budget for concrete execution: 630 ms,  -- 
18:31:23.837 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)1
18:31:24.454 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.propagate(Throwable)
18:31:24.555 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)2
18:31:24.719 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)3
18:31:24.900 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)4
18:31:24.900 | INFO  | Finished (in 1.077 sec): method com.google.common.base.Throwables.propagate(Throwable) 
<StatsForMethod> :
	#generatedTC=4
	WITH NO EXCEPTIONS
18:31:24.925 | DEBUG |  ... 
18:31:24.926 | INFO  | Started: method com.google.common.base.Throwables.throwIfUnchecked(Throwable)
18:31:24.926 | INFO  |  -- Remaining time budget: 22858 ms, #remaining_methods: 18, budget for method: 1269 ms, solver timeout: 634 ms, budget for symbolic execution: 635 ms, budget for concrete execution: 634 ms,  -- 
18:31:24.935 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)1
18:31:25.188 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)2
18:31:25.561 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.throwIfUnchecked(Throwable)
18:31:25.704 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)3
18:31:25.860 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)4
18:31:25.860 | INFO  | Finished (in 0.934 sec): method com.google.common.base.Throwables.throwIfUnchecked(Throwable) 
<StatsForMethod> :
	#generatedTC=4
	WITH NO EXCEPTIONS
18:31:25.862 | DEBUG |  ... 
18:31:25.863 | INFO  | Started: method com.google.common.base.Throwables.getCausalChain(Throwable)
18:31:25.863 | INFO  |  -- Remaining time budget: 21920 ms, #remaining_methods: 17, budget for method: 1289 ms, solver timeout: 644 ms, budget for symbolic execution: 645 ms, budget for concrete execution: 644 ms,  -- 
18:31:26.374 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)1
18:31:26.508 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getCausalChain(Throwable)
18:31:27.153 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getCausalChain(Throwable)
18:31:27.241 | INFO  | Finished (in 1.378 sec): method com.google.common.base.Throwables.getCausalChain(Throwable) 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
18:31:27.265 | DEBUG |  ... 
18:31:27.266 | INFO  | Started: method com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)
18:31:27.266 | INFO  |  -- Remaining time budget: 20518 ms, #remaining_methods: 16, budget for method: 1282 ms, solver timeout: 641 ms, budget for symbolic execution: 641 ms, budget for concrete execution: 641 ms,  -- 
18:31:27.279 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
18:31:27.655 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)1
18:31:27.789 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)2
18:31:27.907 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)
18:31:28.301 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)3
18:31:28.352 | INFO  | Finished (in 1.086 sec): method com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=3
	FAILED 1 time(s) with 1 different exception(s)"
18:31:28.363 | DEBUG |  ... 
18:31:28.364 | INFO  | Started: method com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)
18:31:28.364 | INFO  |  -- Remaining time budget: 19420 ms, #remaining_methods: 15, budget for method: 1294 ms, solver timeout: 647 ms, budget for symbolic execution: 647 ms, budget for concrete execution: 647 ms,  -- 
18:31:28.377 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
18:31:28.416 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)1
18:31:29.011 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)
18:31:29.217 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)2
18:31:29.567 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)3
18:31:29.659 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)
18:31:29.745 | INFO  | Finished (in 1.381 sec): method com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=3
	FAILED 1 time(s) with 1 different exception(s)"
18:31:29.776 | DEBUG |  ... 
18:31:29.777 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)
18:31:29.777 | INFO  |  -- Remaining time budget: 18006 ms, #remaining_methods: 14, budget for method: 1286 ms, solver timeout: 643 ms, budget for symbolic execution: 643 ms, budget for concrete execution: 643 ms,  -- 
18:31:29.783 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
18:31:30.420 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)
18:31:31.064 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)
18:31:31.140 | INFO  | Finished (in 1.363 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class) 
<StatsForMethod> :
	#generatedTC=0
	FAILED 1 time(s) with 1 different exception(s)"
18:31:31.182 | DEBUG |  ... 
18:31:31.182 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class)
18:31:31.183 | INFO  |  -- Remaining time budget: 16601 ms, #remaining_methods: 13, budget for method: 1277 ms, solver timeout: 638 ms, budget for symbolic execution: 639 ms, budget for concrete execution: 638 ms,  -- 
18:31:31.188 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
18:31:31.822 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.propagateIfPossible(Throwable, Class)
18:31:32.461 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.propagateIfPossible(Throwable, Class)
18:31:32.634 | INFO  | Finished (in 1.451 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=0
	FAILED 1 time(s) with 1 different exception(s)"
18:31:32.685 | DEBUG |  ... 
18:31:32.686 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable)
18:31:32.686 | INFO  |  -- Remaining time budget: 15098 ms, #remaining_methods: 12, budget for method: 1258 ms, solver timeout: 629 ms, budget for symbolic execution: 629 ms, budget for concrete execution: 629 ms,  -- 
18:31:32.692 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)1
18:31:32.807 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)2
18:31:33.315 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.propagateIfPossible(Throwable)
18:31:33.380 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)3
18:31:33.513 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)4
18:31:33.699 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)5
18:31:33.700 | INFO  | Finished (in 1.014 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable) 
<StatsForMethod> :
	#generatedTC=5
	WITH NO EXCEPTIONS
18:31:33.724 | DEBUG |  ... 
18:31:33.725 | INFO  | Started: method com.google.common.base.Throwables.getCauseAs(Throwable, Class)
18:31:33.725 | INFO  |  -- Remaining time budget: 14058 ms, #remaining_methods: 11, budget for method: 1278 ms, solver timeout: 639 ms, budget for symbolic execution: 639 ms, budget for concrete execution: 639 ms,  -- 
18:31:33.731 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
18:31:33.837 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)1
18:31:34.249 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)2
18:31:34.347 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)3
18:31:34.365 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getCauseAs(Throwable, Class)
18:31:34.910 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)4
18:31:35.004 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getCauseAs(Throwable, Class)
18:31:35.028 | INFO  | Finished (in 1.302 sec): method com.google.common.base.Throwables.getCauseAs(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=4
	FAILED 1 time(s) with 1 different exception(s)"
18:31:35.063 | DEBUG |  ... 
18:31:35.063 | INFO  | Started: method com.google.common.base.Throwables.getStackTraceAsString(Throwable)
18:31:35.064 | INFO  |  -- Remaining time budget: 12720 ms, #remaining_methods: 10, budget for method: 1272 ms, solver timeout: 636 ms, budget for symbolic execution: 636 ms, budget for concrete execution: 636 ms,  -- 
18:31:35.077 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getStackTraceAsString(Throwable)1
18:31:35.701 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getStackTraceAsString(Throwable)
18:31:35.859 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getStackTraceAsString(Throwable)2
18:31:35.860 | INFO  | Finished (in 0.796 sec): method com.google.common.base.Throwables.getStackTraceAsString(Throwable) 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
18:31:35.902 | DEBUG |  ... 
18:31:35.902 | INFO  | Started: method com.google.common.base.Throwables.lazyStackTrace(Throwable)
18:31:35.903 | INFO  |  -- Remaining time budget: 11881 ms, #remaining_methods: 9, budget for method: 1320 ms, solver timeout: 660 ms, budget for symbolic execution: 660 ms, budget for concrete execution: 660 ms,  -- 
18:31:36.563 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.lazyStackTrace(Throwable)
18:31:37.223 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.lazyStackTrace(Throwable)
18:31:37.224 | INFO  | Finished (in 1.322 sec): method com.google.common.base.Throwables.lazyStackTrace(Throwable) 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
18:31:37.300 | DEBUG |  ... 
18:31:37.301 | INFO  | Started: method com.google.common.base.Throwables.lazyStackTraceIsLazy()
18:31:37.301 | INFO  |  -- Remaining time budget: 10482 ms, #remaining_methods: 8, budget for method: 1310 ms, solver timeout: 655 ms, budget for symbolic execution: 655 ms, budget for concrete execution: 655 ms,  -- 
18:31:37.308 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.lazyStackTraceIsLazy()1
18:31:37.956 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.lazyStackTraceIsLazy()
18:31:38.612 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.lazyStackTraceIsLazy()
18:31:38.623 | INFO  | Finished (in 1.322 sec): method com.google.common.base.Throwables.lazyStackTraceIsLazy() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
18:31:38.710 | DEBUG |  ... 
18:31:38.710 | INFO  | Started: method com.google.common.base.Throwables.jlaStackTrace(Throwable)
18:31:38.711 | INFO  |  -- Remaining time budget: 9073 ms, #remaining_methods: 7, budget for method: 1296 ms, solver timeout: 648 ms, budget for symbolic execution: 648 ms, budget for concrete execution: 648 ms,  -- 
18:31:38.985 | INFO  | Finished (in 0.274 sec): method com.google.common.base.Throwables.jlaStackTrace(Throwable) 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
18:31:39.003 | DEBUG |  ... 
18:31:39.004 | INFO  | Started: method com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
18:31:39.004 | INFO  |  -- Remaining time budget: 8779 ms, #remaining_methods: 6, budget for method: 1463 ms, solver timeout: 731 ms, budget for symbolic execution: 732 ms, budget for concrete execution: 731 ms,  -- 
18:31:39.016 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ReadingFromKryoException: Failure reading from kryo
	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:392) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
Caused by: com.esotericsoftware.kryo.KryoException: Encountered unregistered class ID: 30
	at com.esotericsoftware.kryo.util.DefaultClassResolver.readClass(DefaultClassResolver.java:159) ~[utbot-junit-contest-1.0.jar:?]
	at com.esotericsoftware.kryo.Kryo.readClass(Kryo.java:758) ~[utbot-junit-contest-1.0.jar:?]
	at com.esotericsoftware.kryo.Kryo.readClassAndObject(Kryo.java:869) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.util.KryoHelper.readCommand(KryoHelper.kt:46) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1$1.invokeSuspend(ConcreteExecutor.kt:256) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:84) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1.invoke(ConcreteExecutor.kt:253) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1.invoke(ConcreteExecutor.kt:101) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30) ~[utbot-junit-contest-1.0.jar:?]
18:31:39.520 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])1
18:31:39.737 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
18:31:40.468 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
18:31:40.592 | INFO  | Finished (in 1.587 sec): method com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[]) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
18:31:40.611 | DEBUG |  ... 
18:31:40.612 | INFO  | Started: method com.google.common.base.Throwables.getJLA()
18:31:40.612 | INFO  |  -- Remaining time budget: 7171 ms, #remaining_methods: 5, budget for method: 1434 ms, solver timeout: 717 ms, budget for symbolic execution: 717 ms, budget for concrete execution: 717 ms,  -- 
18:31:41.329 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getJLA()
18:31:42.047 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getJLA()
18:31:42.072 | INFO  | Finished (in 1.460 sec): method com.google.common.base.Throwables.getJLA() 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
18:31:42.122 | DEBUG |  ... 
18:31:42.122 | INFO  | Started: method com.google.common.base.Throwables.getGetMethod()
18:31:42.122 | INFO  |  -- Remaining time budget: 5661 ms, #remaining_methods: 4, budget for method: 1415 ms, solver timeout: 707 ms, budget for symbolic execution: 708 ms, budget for concrete execution: 707 ms,  -- 
18:31:42.152 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getGetMethod()1
18:31:42.831 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getGetMethod()
18:31:43.539 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getGetMethod()
18:31:43.577 | INFO  | Finished (in 1.454 sec): method com.google.common.base.Throwables.getGetMethod() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
18:31:43.673 | DEBUG |  ... 
18:31:43.674 | INFO  | Started: method com.google.common.base.Throwables.getSizeMethod()
18:31:43.674 | INFO  |  -- Remaining time budget: 4110 ms, #remaining_methods: 3, budget for method: 1370 ms, solver timeout: 685 ms, budget for symbolic execution: 685 ms, budget for concrete execution: 685 ms,  -- 
18:31:43.682 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getSizeMethod()1
18:31:44.359 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getSizeMethod()
18:31:45.045 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getSizeMethod()
18:31:45.061 | INFO  | Finished (in 1.387 sec): method com.google.common.base.Throwables.getSizeMethod() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
18:31:45.081 | DEBUG |  ... 
18:31:45.082 | INFO  | Started: method com.google.common.base.Throwables.getJlaMethod(String, Class)
18:31:45.082 | INFO  |  -- Remaining time budget: 2701 ms, #remaining_methods: 2, budget for method: 1350 ms, solver timeout: 675 ms, budget for symbolic execution: 675 ms, budget for concrete execution: 675 ms,  -- 
18:31:45.108 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getJlaMethod(String, Class)1
18:31:45.757 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getJlaMethod(String, Class)
18:31:46.433 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getJlaMethod(String, Class)
18:31:46.434 | INFO  | Finished (in 1.352 sec): method com.google.common.base.Throwables.getJlaMethod(String, Class) 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
18:31:46.536 | DEBUG |  ... 
18:31:46.536 | INFO  | Started: method com.google.common.base.Throwables.<init>()
18:31:46.536 | INFO  |  -- Remaining time budget: 1247 ms, #remaining_methods: 1, budget for method: 1247 ms, solver timeout: 623 ms, budget for symbolic execution: 624 ms, budget for concrete execution: 623 ms,  -- 
18:31:46.541 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.<init>()1
18:31:46.563 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.<init>()2
18:31:46.564 | INFO  | Finished (in 0.027 sec): method com.google.common.base.Throwables.<init>() 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
18:31:46.570 | INFO  | Started: Flushing tests for [Throwables] on disk
18:31:46.871 | INFO  | File size for ThrowablesTest: 34 KB
18:31:46.873 | INFO  | Finished (in 0.303 sec): Flushing tests for [Throwables] on disk 
18:31:46.873 | INFO  | Finished (in 24.568 sec): class com.google.common.base.Throwables 
<StatsForClass> :
	canceled by timeout = false
	#methods = 20, 
	#methods started symbolic exploration = 20
	#methods with at least one TC = 15
	#methods with exceptions = 6
	#generated TC = 36
	#coverage = null
[READY]
